packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -77,12 +104,22 @@ export function Autocomplete({
   async function updateInput(newText: string) {
     setInputText(newText);
     if (newText) {
-      debouncedSetOptions(await getOptions(newText));
+      debouncedSetOptions(mapResultsToOptions(await getOptions(newText)));
     } else {
-      setOptions(initialOptions);
+      setOptions(mapResultsToOptions(initialOptions));
     }
   }
 
+  function mapResultsToOptions(items: AnyOption[]) {
```

Comment:
```suggestion
  function mapToOptions(items: AnyOption[]) {
```

Reply:
This method can also be pulled up to the module level since it doesn't depend on anything within the component.